crates/cxx-qt-lib/src/core/qanystringview.rs

```diff
@@ -0,0 +1,142 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Joshua Goins <josh@redstrate.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+use cxx::{type_id, ExternType};
+use std::ffi::c_void;
+use std::mem::MaybeUninit;
+
+#[cxx::bridge]
+mod ffi {
+    #[namespace = "Qt"]
+    unsafe extern "C++" {
+        include!("cxx-qt-lib/qt.h");
+    }
```

Comment:
this block looks unused? i think you can remove as you aren't using any enums from the Qt namespace?

Reply:
Done